animations/lib/src/basics/01_animated_container.dart

```diff
@@ -55,7 +60,7 @@ class _AnimatedContainerDemoState extends State<AnimatedContainerDemo> {
                     color: color,
                     borderRadius: BorderRadius.circular(borderRadius),
                   ),
-                  duration: _duration,
+                  duration: Duration(milliseconds: 400),
```

Comment:
Ex post facto, unsolicited drive-by comment: 

While moving the constant from the top makes sense, this can still use the const constructor:

```
duration: const Duration(milliseconds: 400),
```